docs/DeveloperGuide.adoc

```diff
@@ -96,48 +98,45 @@ The `UI` component,
 image::LogicClassDiagram.png[]
 
 *API* :
-link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
+link:{repoURL}/src/main/java/seedu/nova/logic/Logic.java[`Logic.java`]
```

Comment:
The same diagram is duplicated above, in the higher-level description of the app's architecture. Perhaps a simpler  and more abstract diagram could be used to describe the higher-level summary of `Logic` (and you can leave the more detailed diagram of  `Logic` here)

Reply:
but I think its good that you provided some notes for the `XYZCommand` (in the class diagram)